packages/core/content-releases/admin/src/components/ReleaseActionMenu.tsx

```diff
@@ -1,49 +1,60 @@
-import { Flex, IconButton, Typography } from '@strapi/design-system';
+import { Flex, IconButton, Typography, Icon } from '@strapi/design-system';
 import { Menu } from '@strapi/design-system/v2';
 import { CheckPermissions, useAPIErrorHandler, useNotification } from '@strapi/helper-plugin';
-import { Cross, More } from '@strapi/icons';
+import { Cross, More, Pencil } from '@strapi/icons';
 import { isAxiosError } from 'axios';
 import { useIntl } from 'react-intl';
+import { useHistory } from 'react-router-dom';
 import styled from 'styled-components';
 
-import { DeleteReleaseAction } from '../../../shared/contracts/release-actions';
+import { DeleteReleaseAction, ReleaseAction } from '../../../shared/contracts/release-actions';
 import { PERMISSIONS } from '../constants';
 import { useDeleteReleaseActionMutation } from '../services/release';
 
-const StyledMenuItem = styled(Menu.Item)`
+const StyledMenuItem = styled(Menu.Item)<{ variant?: 'neutral' | 'danger' }>`
   &:hover {
-    background: ${({ theme }) => theme.colors.danger100};
+    background: ${({ theme, variant = 'neutral' }) => theme.colors[`${variant}100`]};
   }
 
   svg {
     path {
-      fill: ${({ theme }) => theme.colors.danger600};
+      fill: ${({ theme, variant = 'neutral' }) => theme.colors[`${variant}600`]};
     }
   }
 
   &:hover {
```

Comment:
why not group both the hover parts together? avoids someone potentially overwriting something in the future? 😄